README.md

```diff
@@ -4,41 +4,42 @@
 
 [![Join the chat at https://gitter.im/Wayfire-WM/Lobby](https://badges.gitter.im/Wayfire-WM/Lobby.svg)](https://gitter.im/Wayfire-WM/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) or join #wayfire at freenode.net
 
-Wayfire is a wayland compositor based on wlroots. It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance. If you want to gain a better impression at what it can do, see [screenshots](https://github.com/ammen99/wayfire/wiki/Screenshots), [list of plugins](https://github.com/ammen99/wayfire/wiki/Configuration-&-Plugins)
+Wayfire is a wayland compositor based on wlroots. It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance. If you want to gain a better impression at what it can do, see the demo videos on youtube: [link](https://www.youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle)
 
 # Build and install
 
-To build wayfire, you'll need `cmake`, `glm`, `gio-2.0`, `freetype2`, `cairo` and `wlroots` and its build dependencies. Optional dependencies are `gdk-2.0`/`gtk2` and `gdk-pixbuf-2.0`(for loading jpeg backgrounds). When ready, simply clone this repo, compile and install:
+To build wayfire, you'll need `glm`, `wf-config` and`wlroots` + its build dependencies. When ready, simply clone this repo, compile and install:
```

Comment:
Maybe expand to say wf-config is included as a submodule of the project and is built by default if not found, or something?